app/assets/stylesheets/responsive_active_admin/_forms.scss

```diff
@@ -28,10 +28,14 @@ form {
 
       &:not(.has_many_fields) ol {
         @include responsive {
-          width: 80%;
+          padding-left: 0;
         }
       }
     }
+
+    p.inline-hints {
```

Comment:
Please wrap this in a `'include @responsive` mixin (as above), as this currently affects all displays (not just small screens).